### DISCUSSION TITLE
IBM WAS Fixes not longer avaiable for 8.5 - How I do progress ?

### DISCUSSION THREAD

**Discussion by agorosti** [HCL-TECH-SOFTWARE/connections-automation#205] [General]
Hi,
I am trying to run the ansible scripts to install connections, and I am unable to locate all files needed.
Seems ansible scripts to install connections are based on was 8.5,   and they required the fixes fp021  (8.5.5-ws-was-fp021) that are not longer (I am unable to find them) avaible on IBM Fix Central....     
Being an HCL customer I do have an entitlement to use WAS, but the WAS that is shipped on flexnet for connections does not have fixes, so I can not download... 
How I can skip that step and keep on with the quickstart installation ?   I got stuk at :

**TASK [was-nd-fix-install : Install WebSphere Application Server - Fixes] *************************************************
included: /home/ansible/connections-automation/roles/third_party/ibm/wasnd/was-nd-fix-install/tasks/install_was_fixes.yml for connections.giahsa-cloud.com

**Comment by stoeps13** [comment]
You can download all necessary WAS files from Flexnet. Don't skip or workaround the fixes.

Regards
Christoph
